Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -233,8 +233,15 @@ public class PBXProjGenerator {
             .sorted { $0.nameOrPath.localizedStandardCompare($1.nameOrPath) == .orderedAscending }
             .map { $0 }
 
+        let assetTags = Array(Set(project.targets
+            .map { target in
+                target.sources.map { $0.resourceTags }.flatMap { $0 }
+            }.flatMap { $0 }
+        ))
+        
         let projectAttributes: [String: Any] = ["LastUpgradeCheck": project.xcodeVersion]
             .merged(project.attributes)
+            .merged(["knownAssetTags" : assetTags])
```

Comment:
Only add this if `assetTags` is not empty, otherwise we'll cause a diff in the project

Reply:
(This is why the tests are failing by the way)

Reply:
bd567f6